src/Services/CurrencyConverterService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Furniture\Services;
+
+class CurrencyConverterService
+{
+    const VALID_CURRENCIES = ['GBP', 'USD', 'EUR', 'YEN'];
+    const GBP = 1;
```

Comment:
I would have this as `1.0` just in case. If you have an integer price and it gets multiplied by this, your currency converter would return an int, so initialise this to `1.0` just to make sure it's a float.